Scorecounter/app/src/main/java/com/example/anmol/courtcounter/Basketball/BasketballActivity.java

```diff
@@ -370,4 +383,16 @@ public void buttonEnable(){
         undoTeamA.setEnabled(true);
         undoTeamB.setEnabled(true);
     }
+
+    public void addItems(){
+        String title = "Basketball";
+        String outcome = "Result : " + w;
+        String scoreTwo = "[Team A] : " + a + "-" + b + " : [Team B]";
```

Comment:
@Sarthak2601 Please use the Team names given by user instead of using TeamA and TeamB.
Please correct this.....

And do this for all ganes.......

Reply:
@chanmol1999 Okay, creating an issue and working on it.